aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,20 +1,16 @@
 ---
-remote_endpoint: https://fullnode.mainnet.aptoslabs.com
-# replace with below for actual release, compat test needs concrete URL above:
-# remote_endpoint: ~
-name: "vX.YY.Z"
+remote_endpoint: ~
+name: "v1.29.0"
 proposals:
   - name: proposal_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework, version vX.YY.Z"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-vX.YY.Z"
+      title: "Multi-step proposal to upgrade mainnet framework, version v1.29.0"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.29.1-rc"
```

Comment:
The framework usually releases from a different tag (whatever happens to be head at that point so usually just putting the branch name is better)